package.json

```diff
@@ -17,6 +17,7 @@
     "postemulator": "npm run emulator:kill",
     "prerelease": "npm ci && npm run emulator",
     "release": "node release.js",
+    "postrelease": "node postrelease.js &",
```

Comment:
Let's change this to `"node postrelease.js > /dev/null 2>&1 &"`. So it doesn't print anything, otherwise it will log to the console and give the impression the release is still running.